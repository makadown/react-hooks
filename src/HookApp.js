import React from 'react';

const HookApp = props => {
    return (
        <div>
            <h1>Hola Hooks!</h1>
        </div>
    )
}

export default HookApp

