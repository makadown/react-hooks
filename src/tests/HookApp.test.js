import React from 'react';
import { shallow } from 'enzyme';
import HookApp from '../HookApp';

describe('Tests Raices', () => {
    test('Debe cargar ', () => {
        
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    })
    
});
