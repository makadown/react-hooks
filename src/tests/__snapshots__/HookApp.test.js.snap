// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Raices Debe cargar  1`] = `
<div>
  <h1>
    Hola Hooks!
  </h1>
</div>
`;
